std_msgs/Bool trigger
------------------
std_msgs/Bool ifdone
